batcher/aligned/src/main.rs

```diff
@@ -102,6 +110,37 @@ pub struct SubmitArgs {
     keystore_path: Option<PathBuf>,
 }
 
+#[derive(Parser, Debug)]
+#[command(version, about, long_about = None)]
+pub struct DepositToBatcherArgs {
+    #[arg(
+        name = "Batcher Eth Address",
+        long = "conn",
```

Comment:
```suggestion
        long = "payment-service-address",
```

Reply:
It is the same convention in the other commands, I agree tho